tasks/shelter/README.md

```diff
@@ -296,334 +296,334 @@ Pay attention to additions:
+- You don't need to make the header sticky. Those, when scrolling, it remains in its position.
 
-2. Блок **Our Friends**
+2. **Our Friends** section
 
-- Заглавный текст должен быть центрирован. Блок с текстом и отступы могут быть `responsive`.
-- Сетка становится 2 х 3. Блоки с питомцами имеют жесткие размеры, как на макете `assets`. При этом отступы между блоками или краями экрана могут быть `responsive`.
-- Блок с кнопками должен быть центрирован. Размеры кнопок и расстояния между ними заданы жестко, как на `assets`, или как на макете `768px`. Отступы до краев экрана могут быть `responsive`.
+- Heading text should be centered. Block with text and padding can be `responsive`.
+- Grid becomes 2 x 3. Pet blocks are rigidly sized like in the `assets` layout. In this case, the indents between blocks or screen edges can be `responsive`.
+- Block with buttons should be centered. Button sizes and spacings are hardcoded, like on `assets`, or like on the `768px` layout. Padding to the edges of the screen can be `responsive`.
 
-3. **Footer** (`<footer>` содержит контакты, адрес и изображение):
+3. **Footer** (`<footer>` contains contacts, address and image):
 
-- Тут идет сетка из двух колонок. Обратите внимание, что сам блок с сеткой, должен быть центрирован. Т.е. расстояние слева до первой колонки совпадает с расстоянием справа до второй колонки. При этом сами колонки и отступы могут быть `responsive`.
-- Картинка собаки и отступы могут быть `responsive`. Главное, чтобы картинка была центрирована.
+- There is a grid of two columns. Please note that the grid box itself must be centered. Those, the distance to the left of the first column is the same as the distance to the right of the second column. In this case, the columns themselves and indents can be `responsive`.
+- Dog picture and padding can be `responsive`. The main thing is that the picture is centered.
 
 ## 320px <= width < 768px
 
 ### Main Page
 
-1. **Header** (`<header>` содержит только логотип и бургер меню)
+1. **Header** (`<header>` only contains logo and burger menu)
 
-- Меню навигации преобразовывается в так называемое бургер меню, которое будет открываться по нажатию и предлагать привычную нам панель навигации. Логотип дублируется в открытом меню, хотя на макете этого не видно. Однако, если вы сделали так, что меню открывается, а логотип с основной страницы пропадает - это не ошибка, баллы сниматься не будут.
-- Отступ слева от логотипа может быть `responsive`. Отступ справа от меню навигации должен быть задан жестко, как на макете `320px`.
-- Хедер "липким" делать не нужно. Т.е. при скролле он остается на своей позиции.
+- The navigation menu is transformed into the burger menu, which will open on click and offer the familiar navigation bar. The logo is duplicated in the open menu, although this is not visible on the layout. However, if you made the menu open and the logo disappears from the main page - this is not a mistake, points will not be deducted.
```

Comment:
```suggestion
- The navigation menu is transformed into the burger menu, which will open on click and offer the familiar navigation bar. The logo is duplicated in the open menu, although it's not visible on the layout. However, if you implemented it so that the menu opens and the logo disappears from the main page - it's not a mistake, and the score will not be reduced.
```